README.md

```diff
@@ -7,7 +7,7 @@ This is a Python command line project made with [Python](https://www.python.org)
 1. Install [python 3](https://www.python.org)
 2. Install [dependencies](./requirements.txt)
 
-- `pip3 install -r requirements.txt`
+- [`pip3`](https://www.scaler.com/topics/what-is-pip-in-python/) `install -r requirements.txt`
```

Comment:
I do not think this is the best location for adding a hyperlink to pip documentation. This line is meant to be for copy-pasting. You could add a link separately.